.expeditor/end_to_end.pipeline.yml

```diff
@@ -255,4 +255,13 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
             - HAB_BLDR_URL=https://bldr.acceptance.habitat.sh
-  
+
+  - wait
+
+  - label: "[:hab: Promote to Acceptance]"
```

Comment:
When iterating on end-to-end tests, it's common to create one-off builds to validate behavior before merging the PR.  Can/Should we guard against this? Will that have any effect or will it essentially be a no-op?  Should we not be doing one-off builds of the end-to-end pipeline?

Reply:
See https://github.com/habitat-sh/habitat/pull/7043/commits/f2810d7b7df1f30366af802e77c2f8e1f80e3362